src/extension.ts

```diff
@@ -75,6 +75,17 @@ export function activate(context: ExtensionContext) {
 
   function getSettings(): JSON {
     let configXML = workspace.getConfiguration();
+
+    let autoCloseTags = configXML.get("xml.completion.autoCloseTags");
+    let autoClosingBrackets = configXML.get("[xml]")["editor.autoClosingBrackets"];
+    if (autoCloseTags && autoClosingBrackets != "never") {
```

Comment:
you need to ask permission for changing the value. Not do it 1st and ask for forgiveness